ext/intl/php_intl.stub.php

```diff
@@ -278,8 +278,8 @@ function intlcal_get_error_message(IntlCalendar $calendar): string|false {}
  * @param int $hour
  * @param int $minute
  * @param int $second
- * @deprecated
  */
+#[\Deprecated(since: '8.4')]
```

Comment:
```suggestion
#[\Deprecated(message: 'Use IntlGregorianCalendar::__construct(), IntlGregorianCalendar::createFromDate(), or IntlGregorianCalendar::createFromDateTime() instead', since: '8.4')]
```